core/lib/zksync_core/src/state_keeper/io/seal_logic.rs

```diff
@@ -130,12 +130,15 @@ impl UpdatesManager {
 
         let initial_bootloader_contents =
             finished_batch.final_bootloader_memory.unwrap_or_default();
+
+        let events_queue = Vec::new(); // TODO: put actual value when new VM is merged.
```

Comment:
Also, if we want to reduce the diff for the final PR we can integrate returning of the events_queue with the old VM (it will change the interface of the VM, but it won't change its behaviour, so should be fine)

Reply:
I think it's not worth it. When we merge new VM version we will just glue old VM result types into new type, with `events_queue: Vec::new()` the same way it's done currently in `boojum-integration` and it won't require change to old vm versions' implementation. If I do it now then I need to change at least the current latest VM version impl.

Reply:
makes sense

Reply:
@perekopskiy Maybe you could introduce `events_queue: Vec::new()` in all the VM types? Again, as I understand one of the main purposes of this PR is to reduce the diff of `main` compared to the final `boojum-integration` branch. I mean just provide empty vector for the sake of the needed returned type

Reply:
Yes, the main purpose is to reduce the diff. But I don't want to introduce code that won't be needed when the complete boojum integration is merged. We don't need this field in old VM types but only in a new one, which is not in main yet. Also, it will be confusing to have a field that will be always empty